code/modules/flufftext/Hallucination.dm

```diff
@@ -1294,3 +1295,26 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 	H.preparePixelProjectile(target, start)
 	H.fire()
 	qdel(src)
+
+/datum/hallucination/sleeping_carp
+
+/datum/hallucination/sleeping_carp/New(mob/living/carbon/C, forced = TRUE)
+	set waitfor = FALSE
+	..()
+	var/list/mobsyup
+	for (var/mob/living/carbon/A in orange(C,1))
+		if (get_dist(C,A) < 2)
+			LAZYADD(mobsyup,A)
+	if (!LAZYLEN(mobsyup))
+		qdel(src)
+		return
+	var/mob/living/carbon/G = pick(mobsyup)
+	if (rand(0,1))
```

Comment:
should probably be prob(50) for clarity

Reply:
I mean ok. But also. Isn't it clear enough?